test/k8s-integration/main.go

```diff
@@ -657,6 +659,9 @@ func runTestsWithConfig(testParams *testParameters, testConfigArg, reportPrefix
 		kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--log_dir=%s", kubetestDumpDir))
 	}
 	kubeTest2Args = append(kubeTest2Args, "--")
+	if len(*testVersion) != 0 && *testVersion != "master" {
```

Comment:
nit: Maybe we should move all the logic to prepare kubetest2 args under if *useKubeTest2{ ... }, to keep the code clean.

Reply:
We're going to be tearing out kubetest; it's been helpful for debugging to be able to compare args.

Reply:
Ack